components/Players/DraggableQueue.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-list>
+    <draggable v-model="queue" v-bind="dragOptions" class="list-group">
+      <v-hover
+        v-for="(item, index) in queue"
+        :key="`${item.Id}-${getUuid()}`"
+        v-slot="{ hover }"
+        class="pa-0 ma-0"
+      >
+        <v-list-item ripple class="pa-0 ma-0" @click="onClick(index)">
+          <v-list-item-action
+            v-if="!hover"
+            class="list-group-item d-flex justify-center d-flex text-caption"
+            :class="{ 'primary--text': isPlaying(index) }"
+          >
+            {{ index }}
+          </v-list-item-action>
+          <v-list-item-action v-else class="justify-center d-flex">
+            <v-icon>mdi-drag-horizontal</v-icon>
+          </v-list-item-action>
+          <v-list-item-avatar tile class="list-group-item">
+            <blurhash-image :item="item" />
+          </v-list-item-avatar>
+
+          <v-list-item-content>
+            <v-list-item-title
+              :class="{ 'primary--text': isPlaying(index) }"
+              class="text-truncate ml-2 list-group-item"
+            >
+              {{ item.Name }}
+            </v-list-item-title>
+            <v-list-item-subtitle
+              v-if="getArtists(item)"
+              class="ml-2 list-group-item"
+              :class="{ 'primary--text': isPlaying(index) }"
+            >
+              {{ getArtists(item) }}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+
+          <v-list-item-action>
+            <favorite-button :item="item" />
+          </v-list-item-action>
+          <v-list-item-action class="mr-2">
+            <item-menu :item="item" />
+          </v-list-item-action>
+        </v-list-item>
+      </v-hover>
+    </draggable>
+  </v-list>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapActions, mapGetters } from 'vuex';
+import { v4 as uuidv4 } from 'uuid';
+import { BaseItemDto } from '@jellyfin/client-axios';
+
+export default Vue.extend({
+  data() {
+    return {
+      dragOptions: {
+        animation: 500,
+        delay: 0,
+        group: false,
+        dragoverBubble: true,
+        ghostClass: 'ghost'
+      }
+    };
+  },
+  computed: {
+    queue: {
+      get(): BaseItemDto[] {
+        return this.$store.state.playbackManager.queue;
+      },
+      set(newValue: BaseItemDto[]): void {
+        this.setNewQueue({ queue: newValue });
+      }
+    }
+  },
+  methods: {
+    ...mapGetters('playbackManager', ['getCurrentItem']),
+    ...mapActions('playbackManager', ['setNewQueue', 'setCurrentIndex']),
+    isPlaying(index: number): boolean {
+      return index === this.$store.state.playbackManager.currentItemIndex;
+    },
+    getArtists(item: BaseItemDto): string | null {
+      if (item.Artists) {
+        return item.Artists.join(' - ');
```

Comment:
Why a dash and not a simple comma?

Spotify does it well, imo:
![image](https://user-images.githubusercontent.com/19396809/105562195-93ca3400-5d19-11eb-8de6-4b71fb4d5698.png)